photorec, ddrescue: add page (#3625)
tldr-pages/tldr: pages/linux/ddrescue.md

# ddrescue

> > GNU ddrescue is a data recovery tool. It copies data from one file or block device (hard disc, cdrom, etc) to another, trying to rescue the good parts first in case of read errors.  https://www.gnu.org/software/ddrescue/.

Reviewer: This looks like it's from the man page. It's rather verbose - what about this instead?
```suggestion
> Data recovery tool that reads data from damaged block devices.
> More information: <https://www.gnu.org/software/ddrescue/>.
```

> - Make an image (image.dd) of /dev/sdb in the current directory, and make a log:

Reviewer: Perhaps this will read better?
```suggestion
- Take an image of a device, creating a log file:
```

> `sudo ddrescue {{/dev/sdb}} {{image.dd}} {{ddrescue.log}}`

Reviewer: Perhaps this will read better?
```suggestion
`sudo ddrescue {{/dev/sdb}} {{path/to/image.dd}} {{path/to/ddrescue.log}}`
```

> - Clone Disk A to Disk B and make a log:

Reviewer: Perhaps this will read better?
```suggestion
- Clone Disk A to Disk B, creating a log file:
```

> `sudo ddrescue -f -n {{/dev/sda}} {{/dev/sdb}} {{/home/$USER/ddrescue.log}}`

Reviewer: Perhaps this will read better?
```suggestion
`sudo ddrescue --force --no-scrape {{/dev/sda}} {{/dev/sdb}} {{path/to/ddrescue.log}}`
```

# ddrescue

> Data recovery tool that reads data from damaged block devices.
> More information: <https://www.gnu.org/software/ddrescue/>.

- Take an image of a device, creating a log file:

> `sudo ddrescue {{/dev/sdb}} {{path/to/image.dd}} {{path/to/ddrescue.log}}`

Reviewer 2: path/to/log.txt here and after.

Reviewer 2: @einverne - any reason you marked this as resolved ? It doesn't look resolved to me.

Reviewer 3: Oh, sorry, I misunderstood. I was thinking the `{{path/to/ddrescue.log}}` is the modified version. You mean this, right ?

```suggestion
`sudo ddrescue {{/dev/sdb}} {{path/to/image.dd}} {{path/to/log.txt}}`
```

Author: Can I ask why you would rather it be a .txt instead of a .log?  IDC to change it, just my habits have always been to name log files *.log

Reviewer 2: I don't have an issue with the extension. I would like to remove "ddrescue" from the file name since I feel it is redundant, and all file output placeholders that we have are just generic file names. I would be okay with "output.log" also.
